src/main/kotlin/de/bund/digitalservice/useid/tracking/TrackingService.kt

```diff
@@ -0,0 +1,35 @@
+package de.bund.digitalservice.useid.tracking
+
+import mu.KotlinLogging
+import org.springframework.stereotype.Service
+import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+
+@Service
+class TrackingService(private val trackingProperties: TrackingProperties) {
+
+    private val log = KotlinLogging.logger {}
+    private val httpClient: HttpClient = HttpClient.newBuilder().build()
+    private val responseHandler: HttpResponse.BodyHandler<String> = HttpResponse.BodyHandlers.ofString()
+
+    private fun sendTrackingRequest(url: String) {
+        val request = HttpRequest.newBuilder()
+            .uri(URI.create(url))
+            .build()
+
+        val response = httpClient.send(request, responseHandler)
+        val status = response.statusCode()
+        if (status != 200) {
+            log.error("$status, tracking failed for: $url")
+        }
+        log.info("$status, successfully tracked: $url")
+    }
+
+    fun sendMatomoEvent(category: String, action: String, name: String) {
+        val matomoSiteId = trackingProperties.matomoSiteId
+        val domain = trackingProperties.matomoDomain
+        sendTrackingRequest("$domain?idsite=$matomoSiteId&rec=1&ca=1&e_c=$category&e_a=$action&e_n=$name")
```

Comment:
Are we only tracking events? Weren't three of them page views?
And maybe we also want to track user agent or language?
Maybe Max can clarify, what is best, and answer those questions.

Reply:
Sent this to Max and he will get back to us. 
However, we are not having any tickets for tracking more things than events. 
I would vote for us to track events first and increase tracking activity with a new ticket.

Reply:
We would for now only track events, not pageview. We can convert events into goals in matomo. that would allow us to to have events, as often as they occur, but goals only once per user. Which is a great advantage because this allows to answer the question "how many users struggled" or "how many users succeeded" way more accurately. I hope that makes sense?

Reply:
user agent and language are for now not of interest since we only identified the few main KPIs. We try to work according to the principle to only track the must haves for now. anything else will be subject for improvements since we do not want to get any negative feedback from BSI, BMI or the community. Any dimension we identify along the road must be explicitly requested with a clear motivation for tracking

Reply:
For me it doesn't matter. And if you say, we track events, then events it is! :) 

> user agent and language are for now not of interest since we only identified the few main KPIs. We try to work according to the principle to only track the must haves for now. anything else will be subject for improvements since we do not want to get any negative feedback from BSI, BMI or the community. Any dimension we identify along the road must be explicitly requested with a clear motivation for tracking

Really nice, fully agree!